query-connector/src/app/fhir-servers/fhirServers.module.scss

```diff
@@ -0,0 +1,62 @@
+.fhirServersRow {
+  grid-template-columns: 1fr 2fr 2fr;
+  vertical-align: middle;
+  margin: 0 !important;
+  padding: 0 1.5rem;
+  td {
+      padding: 1rem;
+      background-color: transparent;
+    }
+}
+
+.mainContainer {
+  max-width: 80%;
+  padding: 2rem;
+}
+
+.titleContainer {
+  max-width: 100%;
+  margin-left: none;
+  margin-right: none;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.lastChecked {
+  color: #71767A;
+  margin-left: 0.25rem;
+  margin-left: 0.5rem;
+  font-size: 0.875rem;
+}
+
+.tableRowHover {
+  transition: background-color 0.2s ease;
+  border-collapse: collapse;
+
+  &:hover,
+  &:hover > td {
+    background-color: rgba(217, 232, 246, 0.50);
```

Comment:
you should be able to apply the background to just the `&:hover` parent, and then the `td` children will inherit it (see [here](https://stackoverflow.com/questions/27332016/how-do-i-layer-css-opacity) for more of an explanation)

that seems to be working when I adjust it on my local, but I know I ran into a similar issue before where the child elements weren't picking it up, and I had to do some math-y magic (like, parent has 0.5, child has 0.125) - lmk if the above doesn't do the trick!

Reply:
Amazing, this worked!! Thanks @katyasoup!!